XTS/ssl/xts-over-ssh.sh

```diff
@@ -28,7 +28,8 @@ SLEEP_TIME=`timeout_adjust 3 2>/dev/null || echo 3` # default sleep time is 3
 
 WORKSPACE=${WORKSPACE:-$PWD}
 WILDFLY_QUICKSTART_HOME=${WILDFLY_QUICKSTART_HOME:-$QUICKSTART_HOME}
-JBOSS_BIN=${JBOSS_BIN:-$JBOSS_HOME} # jboss_bin is unpacked jboss distribution
+#comment so main wildfly will be used (TODO: uncomment it as soon as narayana works with main wildfly)
```

Comment:
I think you should raise a JBTM to get this line reverted to help remind us to revert this soon. You should reference the JBTM in the TODO comment.

Reply:
Thank you Tom, I think it is a good idea. See the issue here: https://issues.redhat.com/browse/JBTM-3679